sway-lib-std/src/u256.sw

```diff
@@ -311,19 +303,19 @@ impl core::ops::Multiply for U256 {
         let one = ~U256::from(0, 0, 0, 1);
 
         let mut total = zero;
-        // The algorithm loops <from number of bits - 1> to <zero>.
-        // Need to add 1 here to invalidate the while loop once i == 0 since we
-        // don't have a break keyword.
-        let mut i = 256 - 1 + 1;
-
-        while i > 0 {
-            // Workaround for not having break keyword
-            let shift = i - 1;
+
+        let mut i = 256 - 1;
+
+        while i >= 0 {
```

Comment:
`i` is unsigned, so this will always be true.  The `break` below is always ending the loop so this might as well be `while true` to avoid ambiguity.  Personally I'd prefer a var name like `bit_idx` rather than `i` too.
```suggestion
        while true {
```